src/nvim/ex_cmds.c

```diff
@@ -4832,7 +4832,6 @@ void ex_helptags(exarg_T *eap)
           EW_FILE|EW_SILENT) == FAIL
       || filecount == 0) {
     EMSG2("E151: No match: %s", NameBuff);
-    xfree(dirname);
```

Comment:
We need to free `dirname` here. It is an allocated string (see line 4815). The vim-patch tried to delete a function argument `dirname`. So I would say this patch is `NA`.